base_classes/NXdata.nxdl.xml

```diff
@@ -219,29 +219,32 @@
 		
 		There are two older methods of associating 
 		each data dimension to its respective dimension scale.
-		Both are now out of date and
-		should not be used when writing new data files.
-		However, client software should expect to see data files
+		Both should not be used when writing new data files, but
+	    client software should expect to see legacy data files
 		written with any of these methods.
 		
-		  * One method uses the ``axes`` 
-		    attribute to specify the names of each *dimension scale*.
+		  * One method adds the ``axes`` attribute to the data, rather than to
+		    the group. This is not recommended because the data field could be
+		    linked from another group, which does not contain all the 
+		    *dimension scales*.
 		
-		  * The oldest method uses the ``axis`` attribute on each
-		    *dimension scale* to identify
-		    with an integer the axis whose value is the number of the dimension.
+		  * The oldest method adds am ``axis`` attribute to each
+		    *dimension scale* set to an integer corresponding to the 
+		    corresponding dimension.
 	</doc>
-	<field name="VARIABLE" type="NX_NUMBER" nameType="any">
+	<field name="AXIS" type="NX_NUMBER" nameType="any">
 		<doc>
-	 		Dimension scale defining an axis of the data.
+	 		Dimension scale defining an axis of the data. There should be one
+	 		of these for each dimensions of the ``DATA`` field.
```

Comment:
typo: `dimensions` should be singular

Reply:
Thanks for spotting the typos. I think they are now fixed.